### PR TITLE
NONE: Update eslint

### PR DESCRIPTION
**Pull request by atraversatlassian**
**What's in this PR?**
Updates eslint to latest version.  Disables new rules to allow gradual introduction and more readable PRs

**Why**
Keep up with best practices

**Added feature flags**
None

**Affected issues**  
NONE

**How has this been tested?**  
Tests

**Whats Next?**
More lint

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -134,5 +133,29 @@
 			"@typescript-eslint/no-unsafe-return": "off",
 			"@typescript-eslint/no-unsafe-call": "off"
 		}
-	}]
+	},
+	{
+		"files": ["src/**/*.ts", "test/**/*.ts"],
+		"rules": {
+			// TODO: Re-enable gradually
```
Comment (yvonnevictoria) [inline .eslintrc.json:140]: How are we going to keep track of this TODO? Is it worth creating a ticket/assigning an owner?

File: spa/tsconfig.json [hunk 2]
```diff
@@ -24,3 +23,6 @@
 	},
   "include": ["src"]
 }
+
+// FAIL src/pages/ConfigSteps/test.tsx
```
Comment (yvonnevictoria) [inline spa/tsconfig.json:27]: Is this meant to be here?

**Review by yvonnevictoria — DISMISSED** [review]

File: .eslintrc.json (hunk 1 above)
Comment (atraversatlassian) [reply .eslintrc.json:140]: I can create a ticket if you want but just been working through the ESLint as I find time.  The TODO is mostly to mark that these rules aren't here to disable them long term

File: spa/tsconfig.json (hunk 2 above)
Comment (atraversatlassian) [reply spa/tsconfig.json]: This was cleaned up it was not meant to be here

**Review by yvonnevictoria — APPROVED** [review]

**Pull request merged**